x/inter-tx/keeper/msg_server.go

```diff
@@ -30,20 +31,29 @@ func (k msgServer) Register(
 	// check if an account is already registered
 	_, err = k.GetIBCAccount(ctx, msg.SourcePort, msg.SourceChannel, acc)
 	if err == nil {
-		return &types.MsgRegisterAccountResponse{}, fmt.Errorf("Interchain account is already registered for this account")
+		err = fmt.Errorf("Interchain account is already registered for this account")
+		return &types.MsgRegisterAccountResponse{}, sdkerrors.Wrap(err, err.Error())
```

Comment:
All errors need to be registered otherwise the error message gets wiped downstream. The first error cannot begin with 1 either. Usually these errors are registered in `errors.go`

Reply:
Updated this thanks